### PR TITLE
Update gitian-building.md

### PR DESCRIPTION
**Pull request by miketwenty1**
fixing typo

### CODE REVIEW

File: gitian-building.md [hunk 1]
```diff
@@ -124,6 +124,6 @@ This will create the `.sig` files that can be committed together with the `.asse
 Gitian build.
 
 
- `./gitian-build.py --detach-sign -s $NAME $VERSION --nocommit`
+ `./gitian-build.py --detach-sign -s $NAME $VERSION --no-commit`
```
Comment (hebasto) [inline gitian-building.md:127]:
```suggestion
 `./gitian-build.py --detach-sign --no-commit -s $NAME $VERSION`
```

to be consistent with the previous
https://github.com/bitcoin-core/docs/blob/93dc9ff1b529276e78e6f031c786b774997b39bc/gitian-building.md#L87

**Review by hebasto — CHANGES_REQUESTED** [review]
Concept ACK 93dc9ff1b529276e78e6f031c786b774997b39bc.

Also I'd suggest to use long versions of arguments in documentation, i.e.:
- s/`-b`/`--build`/
- s/`-s`/`--sign`/

**Comment by hebasto** [comment]
LGTM, mind [squashing](https://github.com/bitcoin/bitcoin/blob/master/CONTRIBUTING.md#squashing-commits) two commits into single one?

**Comment by miketwenty1** [comment]
@hebasto np, done.

**Comment by jonasschnelli** [comment]
Verified (https://github.com/bitcoin/bitcoin/blob/master/contrib/gitian-build.py#L171).
utACK aafd085961bcdf4c0cd1efb6c823c63c37a07a02

**Review by hebasto — APPROVED** [review]
ACK aafd085961bcdf4c0cd1efb6c823c63c37a07a02, I have reviewed the code and it looks OK, I agree it can be merged.

**Pull request merged**